lib/model/model.go

```diff
@@ -2870,12 +2870,6 @@ func (m *model) blockAvailabilityRLocked(cfg config.FolderConfiguration, file pr
 	return candidates
 }
 
-func (m *model) fileAvailability(cfg config.FolderConfiguration, file protocol.FileInfo) []Availability {
```

Comment:
If you're deleting this, why not delete the function it calls too: `fileAvailabilityRLocked()` right after it?